src/parser/tests/expression/AssociativeArrayLiterals.spec.ts

```diff
@@ -49,15 +49,15 @@ describe('parser associative array literals', () => {
                 token(TokenKind.LeftCurlyBrace, '{'),
                 identifier('foo'),
                 token(TokenKind.Colon, ':'),
-                token(TokenKind.IntegerLiteral, '1', new Int32(1)),
+                createToken(TokenKind.IntegerLiteral, '1'),
```

Comment:
Why using sometimes `token` and sometimes `createToken`?

Reply:
That was an oversight on my part. the `token()` function is a test helper function, and `createToken` is from `astUtils/creators.ts`. I'll go through and change all of these tests back to `token` since there's no functional difference. Thanks!